sonar-scanner-engine/src/main/java/org/sonar/scanner/issue/ModuleIssues.java

```diff
@@ -94,32 +89,33 @@ public boolean initAndAddIssue(Issue issue) {
     ScannerReport.Issue rawIssue = builder.build();
 
     if (filters.accept(inputComponent.key(), rawIssue)) {
-      write(component, rawIssue);
+      write(inputComponent.batchId(), rawIssue);
       return true;
     }
     return false;
   }
 
-  private void applyFlows(ScannerReport.Issue.Builder builder, ScannerReport.IssueLocation.Builder locationBuilder, ScannerReport.TextRange.Builder textRangeBuilder, Issue issue) {
+  private static void applyFlows(ScannerReport.Issue.Builder builder, ScannerReport.IssueLocation.Builder locationBuilder, ScannerReport.TextRange.Builder textRangeBuilder, Issue issue) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Split this 187 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00103)